app/javascript/dashboard/components/widgets/conversation/helpers/botMessageContentHelper.js

```diff
@@ -38,20 +36,52 @@ const generateFormContent = (
   return '';
 };
 
+const generateCSATContent = (
+  contentAttributes,
+  { ratingTitle, feedbackTitle }
+) => {
+  const {
+    submitted_values: { csat_survey_response: surveyResponse = {} } = {},
+  } = contentAttributes;
+  const { rating, feedback } = surveyResponse || {};
+
+  let messageContent = '';
+  if (rating) {
+    const [ratingObject = {}] = CSAT_RATINGS.filter(
+      csatRating => csatRating.value === rating
+    );
+    messageContent += `<div><strong>${ratingTitle}</strong></div>`;
+    messageContent += `<p>${ratingObject.emoji}</p>`;
+  }
+  if (feedback) {
+    messageContent += `<div><strong>${feedbackTitle}</strong></div>`;
```

Comment:
We could use multiline support of template literals here. which is gives more readable html.

```suggestion
    messageContent += `
      <div>
         <strong>${feedbackTitle}</strong>
       </div>
       <p>${ratingObject.emoji}</p>
`;
```

Reply:
Also a `/n` at the end to add some spacing between timestamp and feedback content.

Reply:
<img width="506" alt="Screenshot 2021-06-25 at 1 36 09 PM" src="https://user-images.githubusercontent.com/2246121/123392103-4dc01780-d5ba-11eb-97da-47905e765b5c.png">

It would mess up the spec with weird spacing issues. Ideally, this logic should have been a component rather than a function. That refactor is out of scope for this PR.